### PR TITLE
Submit new gentoo package: zsh-theme-powerlevel9k

### PR DESCRIPTION
**Pull request by Fulgurance**
Hello, i'm new proxy maintainer for new ebuild app-shells/zsh-theme-powerlevel9k package. This package install zsh powerlevel 9k theme for zsh.

Bugzilla: https://bugs.gentoo.org/671514
My Overlay: https://github.com/Fulgurance/fulgurance-overlay
Signed-off-by: zohran.londais@gmail.com

### CODE REVIEW

File: app-shells/zsh-theme-powerlevel9k/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/metadata.xml]: Add proxy maintainers project last, as you are not a gentoo developer

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: copyright line is wrong

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Powerlevel9k theme for ZSH which uses Powerline Fonts."
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: no dot in the `DESCRIPTION`

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+HOMEPAGE="https://github.com/bhilburn/powerlevel9k"
+KEYWORDS="~amd64"
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: wrong bariables order, `KEYWORDS` should not be here

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+LICENSE="MIT"
+SLOT="0"
+RDEPEND="media-fonts/powerline-symbols"
+SRC_URI="https://github.com/bhilburn/powerlevel9k/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: it should go down and each variable spearated by empty lines, see other ebuilds

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+EXTRACT_DIR="powerlevel9k-0.6.6"
+
+src_install()
+{
+	insinto "/usr/share/zsh-theme-powerlevel9k"
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: you do not need quotes

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+	doins -r {"${EXTRACT_DIR}/debug","${EXTRACT_DIR}/docker","${EXTRACT_DIR}/functions","${EXTRACT_DIR}/shunit2","${EXTRACT_DIR}/test","${EXTRACT_DIR}/test-bsd-vm","${EXTRACT_DIR}/test-vm","${EXTRACT_DIR}/test-vm-providers"}
+	doins "${EXTRACT_DIR}/CHANGELOG.md"
+	doins "${EXTRACT_DIR}/CODE_OF_CONDUCT.md"
+	doins "${EXTRACT_DIR}/LICENSE"
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: we do not install license files

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+SLOT="0"
+RDEPEND="media-fonts/powerline-symbols"
+SRC_URI="https://github.com/bhilburn/powerlevel9k/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"
+EXTRACT_DIR="powerlevel9k-0.6.6"
+
+src_install()
+{
+	insinto "/usr/share/zsh-theme-powerlevel9k"
+	doins -r {"${EXTRACT_DIR}/debug","${EXTRACT_DIR}/docker","${EXTRACT_DIR}/functions","${EXTRACT_DIR}/shunit2","${EXTRACT_DIR}/test","${EXTRACT_DIR}/test-bsd-vm","${EXTRACT_DIR}/test-vm","${EXTRACT_DIR}/test-vm-providers"}
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: this is pretty ugly, if there is no better way of handeling this, you can use the `for` loop

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+	doins "${EXTRACT_DIR}/CHANGELOG.md"
+	doins "${EXTRACT_DIR}/CODE_OF_CONDUCT.md"
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: I do not see how this file is useful for the end user

File: app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+	doins "${EXTRACT_DIR}/LICENSE"
+	doins "${EXTRACT_DIR}/powerlevel9k.zsh-theme"
+	doins "${EXTRACT_DIR}/prompt_powerlevel9k_setup"
+	doins "${EXTRACT_DIR}/README.md"
+	doins "${EXTRACT_DIR}/test-in-docker"
+	doins "${EXTRACT_DIR}/TESTS.md"
+}
+
+pkg_postinst() {
+	elog "To use powerlevel9k zsh theme, add 'source \"/usr/share/zsh-theme-powerlevel9k/powerlevel9k.zsh-theme\"' to your .zshrc file"
```
Comment (Zlogene) [inline app-shells/zsh-theme-powerlevel9k/zsh-theme-powerlevel9k-0.6.6.ebuild]: it looks like it needs better formatting

**Comment by Zlogene** [comment]
Also your commit message should comply GLEP66 for new packages ( check it here: https://www.gentoo.org/glep/glep-0066.html)

File: app-shells/zsh-theme-powerlevel9k/metadata.xml (hunk 1 above)
Comment (Fulgurance) [reply app-shells/zsh-theme-powerlevel9k/metadata.xml]: Sorry, but i just don't understand this problem...(i'm french sorry).

**Pull request closed without merging**